book/src/getting-started/3-exposing-to-qml.md

```diff
@@ -26,16 +26,22 @@ There are two notable changes compared to a normal Qt application though:
 {{#include ../../../examples/qml_minimal/cpp/main.cpp:book_qml_register}}
 ```
 
-For every QObject subclass that we define in Rust, CXX-Qt will generate a corresponding C++ class.
-This class is included by the first code snippet.
-They will always be in the `cxx-qt-gen/include/` include path and use the snake_case naming convention.
+For every `#[cxx_qt::bridge]` that we define in Rust, CXX-Qt will generate a corresponding C++ header file.
```

Comment:
this is sortof correct, note how we need to tell our parser that files to search in the `build.rs`, but this is explained in "Building with {CMake/Cargo}` so maybe it's fine to omit here?

Reply:
I think it's fine to omit.
Readers don't need to worry about that part yet.